tools/start_test.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
Let's figure out a way to combine start_test.sh and start_tmux.sh. Maybe pass in a variable with `make start_server` for starting all servers and `make start_test` for starting all tests